tests/unit/test_utils.py

```diff
@@ -1223,6 +1225,29 @@ def test_path_to_url_win():
     assert path_to_url('file') == 'file:' + urllib_request.pathname2url(path)
 
 
+@pytest.mark.parametrize('netloc, has_port, url', [
+    # Test domain name.
+    ('example.com', False, 'https://example.com'),
```

Comment:
I think these would be easier to read and compare these values with the second and third items reversed. Also, `build_url_from_netloc()` is the lower-level (more inner of the two functions), so it makes additional sense to test and display that value first.